lib/Migrator/Migrator.cpp

```diff
@@ -439,6 +439,5 @@ const MigratorOptions &Migrator::getMigratorOptions() const {
 const StringRef Migrator::getInputFilename() const {
   auto &PrimaryInput = StartInvocation.getFrontendOptions()
                            .Inputs.getRequiredUniquePrimaryInput();
-  assert(!PrimaryInput.file().empty());
```

Comment:
What's up with the removal of this assertion? (Just wondering.)

Reply:
InputFiles now contain that assertion on both setter and getters.